component/service/src/main/java/org/exoplatform/social/service/malwareDetection/rest/MalwareDetectionRestService.java

```diff
@@ -27,22 +28,30 @@
 @Api(tags = "/malware", value = "/malware", description = "Malware detection")
 public class MalwareDetectionRestService implements ResourceContainer {
 
+
+  private static final String MALWARE_DETECTION_PASSPHRASE_PROPERTY = "exo.malwareDetection.passphrase";
+  private static final String MALWARE_DETECTION_PASSPHRASE_PROPERTY_DEFAULT = "@malwareDetection/TOCHANGE!@";
```

Comment:
Should we put a default value ? 

In fact, if someone forget to change that, the service is accessible with the defualt passphrase.
If we dont put a default passphrase, and do not let the access to the service is the passphrase is not configured, it is more secured.